resources/js/components/forms/index.js

```diff
@@ -41,3 +42,6 @@ import ToggleSwitchInput from './ToggleSwitchInput.vue'
 Vue.component('SignatureInput', () => import('./SignatureInput.vue'))
 Vue.component('RichTextAreaInput', () => import('./RichTextAreaInput.vue'))
 Vue.component('DateInput', () => import('./DateInput.vue'))
+
+// Register the PhoneInput component globally after it's loaded
+Vue.component('PhoneInput', PhoneInput);
```

Comment:
Please respect how other lazy components are imported, no need for a comment here:
`Vue.component('PhoneInput',()=> import('./PhoneInput.vue'))`